app/controllers/v1/nominations_controller.rb

```diff
@@ -0,0 +1,37 @@
+class V1::NominationsController < V1::BaseController
+  FORM_ID = '1sQtLTyZWA6KDsi7-ToB2VyOGaz44MZEHQ_fUvlX8VQ0'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.